spec/factories/task_managements.rb

```diff
@@ -1,10 +1,15 @@
 FactoryGirl.define do
   factory :task_management do
-    start_time Time.now
-    end_time 2.days.from_now
     task_id 1
     tasker_id 1
     taskee_id 2
-    amount 5000
+    task_desc "A task"
```

Comment:
consider using https://github.com/stympy/faker#fakerlorem for the description. Also, use Faker for generic attributes.